### DISCUSSION TITLE
Possibility to create a fancy custom group node?

### DISCUSSION THREAD

**Discussion by WatsonFungHK** [xyflow/xyflow#2831] [Q&A]
![Screenshot 2023-02-13 at 5 04 42 PM](https://user-images.githubusercontent.com/26916566/218415721-f19c65c2-4408-4903-a464-b62d22e36a99.png)
I am going to make a node similar to the screenshot.

I encounter two issues
1. child node could not link to another node
2. cannot get rid of styling of original group node, e.g. styling upon hover/selected

why is it so difficult to make fancy group node? Is there any way to make it?

**Comment by moklick** [comment]
1. What do you mean with "child node could not link to another node" ? Are you using the "parentNode" option ? 
2. You can get rid of the styling by importing "reactflow/dist/base.css" instead of "reactflow/dist/style.css"

**Reply by WatsonFungHK** [reply]
> 1. What do you mean with "child node could not link to another node" ? Are you using the "parentNode" option ?
> 2. You can get rid of the styling by importing "reactflow/dist/base.css" instead of "reactflow/dist/style.css"

1. Yes, I am using parentNode which I put the id of the parent node. I want to render the child nodes inside the parent node with fixed positions. and each of the nodes could link to other node. 
2. great. kind of solved my current problem
